<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="QcheckOriImg">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Origin Image</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="QcheckImgGt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Image with Gt</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="QcheckPredImg">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Image with Pred Box</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QCal">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>calculate performance</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLResults">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Results not calculated yet!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
